app/DTO/Rights/InitRightsDTO.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\DTO\Rights;
+
+use App\DTO\ArrayableDTO;
+
+/**
+ * This DTO provides the information whether some actions are available to the user.
+ */
+class InitRightsDTO extends ArrayableDTO
+{
+	public function __construct(
+		public RootAlbumRightsDTO $root_album,
+		public SettingsRightsDTO $settings,
+		public UserRightsDTO $users
+	) {
+	}
+
+	/**
+	 * Create from current user.
+	 *
+	 * @return self
+	 */
+	public static function ofCurrentUser(): self
+	{
+		return new self(
+			root_album: RootAlbumRightsDTO::ofCurrentUser(),
+			settings: SettingsRightsDTO::ofCurrentUser(),
+			users: UserRightsDTO::ofCurrentUser()
+		);
+	}
+
+	/**
+	 * Create from no admin registerd.
+	 *
+	 * @return self
+	 */
+	public static function ofAdminIsNotRegistered(): self
+	{
+		return new self(
+			root_album: RootAlbumRightsDTO::ofTrue(),
```

Comment:
`ofTrue` isn't a good name either. Rights `ofCurrentUser`, `ofAlbum`, etc. all make sense, but a sentence like "rights of (the) true" in analogy to "rights of (the) current user" is not even correct with respect to grammar.

Again, I understand where you come from, but the name should be something like

```suggestion
			root_album: RootAlbumRightsDTO::ofUnregisteredAdmin(),
```

Reply:
Fixed [31a5e2b](https://github.com/LycheeOrg/Lychee/pull/1539/commits/31a5e2b97c5302595e7b13326c54d6980c16498b)